crypto/verifier_test.go

```diff
@@ -81,3 +82,59 @@ func TestSignVerify(t *testing.T) {
 		}
 	}
 }
+
+func TestSignVerifyObject(t *testing.T) {
+	key, err := keys.NewFromPrivatePEM(testonly.DemoPrivateKey, testonly.DemoPrivateKeyPass)
+	if err != nil {
+		t.Fatalf("Failed to open test key, err=%v", err)
+	}
+	signer := NewSHA256Signer(key)
+
+	type subfield struct {
+		c int
+	}
+
+	for _, tc := range []struct {
```

Comment:
Suggestion: You could use a []interface{} directly, given that the struct has only one field.

There's an argument for keeping the struct as well, so feel free to ignore.

`for _, tc := range []interface{...} { ... }`